gnucash/gnome/dialog-vendor.c

```diff
@@ -713,9 +713,9 @@ gnc_vendor_search (GtkWindow *parent, GncVendor *start, QofBook *book)
     /* Build parameter list in reverse order */
     if (params == NULL)
     {
-        params = gnc_search_param_prepend (params, _("Billing Contact"), NULL, type,
+        params = gnc_search_param_prepend (params, _("Payment Contact"), NULL, type,
```

Comment:
In accounting 'Billing' is not the same as 'Payment'. I think this is not a good substitute. I see however this is also used in the 'New Vendor' dialog window, so this may need some more discussion.

Reply:
In new vendor dialog, there's no field named "Billing Contact", only "Payment Address Name"

In find vendor dialog, one option is "Billing contact"

They describe one field in different strings.

So I do these: 
Rename "Name" to "Contact", "Billing contact" to "Payment Contact"

Reply:
@gjanssens 
Hi, sir, below are the screenshots, before and after changes. 
Would these changes be ok, or you give me some suggestions? 

customer
[Before](https://github.com/TianXing-Yi/GImage/blob/master/2.0-Before.png)
[After](https://github.com/TianXing-Yi/GImage/blob/master/2.0-After.png)

vendor
[Before](https://github.com/TianXing-Yi/GImage/blob/master/2.1-Before.png)
[After](https://github.com/TianXing-Yi/GImage/blob/master/2.1-After.png)

employee
[Before](https://github.com/TianXing-Yi/GImage/blob/master/2.2-Before.png)
[After](https://github.com/TianXing-Yi/GImage/blob/master/2.2-After.png)

Reply:
As I said `Payment information` is not the proper term IMO. There is no such thing as a `Payment contact` or a `Payment address`. The only payment related element in all of the vendor options are the `Payment terms`.

So instead of changing `Billing` to `Payment` everywhere, you should go the other way around.

Reply:
Get it.

Reply:
@gjanssens, payment for vendors and employees, billing for customers.

Reply:
@jralls Why ? It sound very weird to me  to have an address related to a payment. That's why I suggested Vendor/Customer/Employee address in the followup design.

Reply:
Why does it seem weird to have an address for payment? If you were to still pay bills with paper checks sent in the post you have to put an address on the envelope. 

Doesn't it seem weird to you to (send a) bill (to) a vendor?

Reply:
I probably don't fully understand the term "billing" then.

Regardless however, my later point is that I don't think we should specialize the address type on anything if we only have one address. It's just the vendor's address, right ?
For customer there's a provision for an optional shipping address. So  some specialization is  needed. But for me it still is comfortable to call the main address simply the customer address and  keep shipping address as an optional extra specification. All customers have *an* address, but not all of them have a separate shipping address.

What would you propose as alternative for "Billing/Payment information" for vendors? For me "Payment Information" doesn't cover what's on that tab. A payment term, yes, that's payment information, but that's so both for customers and vendors. Currency is more than just payment information. It defines a property of all  amounts on the bill.  Of course that also impacts the payment,  but I'm not happy just calling it payment information. Similarly Tax details are an accounting property of the bill, that has among others an effect on the total amount. But I hope  you see my point: I think payment information is too narrow. Thinking about this "Bill/Invoice/Voucher defaults" probably comes closer - all but currency can be overridden on a per bill basis.

Reply:
The tab holds the customized Billing Terms.

Reply:
> Why does it seem weird to have an address for payment? If you were to still pay bills with paper checks sent in the post you have to put an address on the envelope.
> 
> Doesn't it seem weird to you to (send a) bill (to) a vendor?

In Europe paper cheques are now rarely used. The IBAN is the primary address of payments followed by BIC+account-number.

Retranslated I would call it the (msgcontext=general) invoice address.

Reply:
Maybe it would help to know that "Bill" and "Invoice" are essentially synonymous in this usage. A customer has a billing address because you send them a bill demanding payment and a vendor has a payment address because you'll be sending them a payment, not a bill.

The Vendor and Customer Edit dialogs are correct as they are now: The Vendor one says Payment Address and Payment Information, the Customer one says Billing Address and Billing Information; Customer has an additional Shipping Address tab.

I don't see any problem with changing Billing/Payment Address to just Address *in English* but I think that they'll need translation context to make them separate msgids because it's possible that they'd need different translations in some languages. Changing the Payment/Billing Information tabs to just Information would be confusing even in English.

Reply:
I agree with gjanssens's point of view. 

For "Billing/Payment Address", the suitable labels may be "Bank Identifier Code, IBAN, Address of bank, Account".

According to existing labels, "Vendor Address" will be better than "Payment Address".